apps/discovery_api/mix.exs

```diff
@@ -50,7 +50,7 @@ defmodule DiscoveryApi.Mixfile do
       {:csv, "~> 2.3"},
       {:credo, "~> 1.0", only: [:dev, :test, :integration], runtime: false},
       {:checkov, "~> 1.0", only: [:test, :integration]},
-      {:divo, "~> 1.1", only: [:dev, :test, :integration]},
```

Comment:
same as alchemist, missing mix increment of app version number. since divo was updated in disc_api, disc_api needs a new mix version